src/Render/Markdown/TemplateRenderer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Scriban;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace D2L.Dev.Docs.Render.Markdown {
+	internal static class TemplateRenderer {
+	
+		public static async Task<string> RenderAsync( 

```

Comment:
Does this need its own class? I don't know if we will be adding any complexity aside from this basic function.

Reply:
I would make it take the template as a string in the constructor and not have this be a static method. We shouldn't need to read the template each time.

Having it as its own non-static class makes sense, in theory, for testability.

Reply:
Sounds good.